docs/developer/app-store/overview.mdx

```diff
@@ -23,7 +23,7 @@ Majority of Saleor Apps are [open source](https://github.com/saleor/apps). Feel
 
 Saleor App Store is a [Saleor Cloud](/cloud/overview.mdx) feature that provides an instantly installable collection of apps. Each app brings new functionality to Saleor, often by integrating with third-party services.
 
-You can find Saleor App Store under the _Apps_ tab in the Dashboard menu.
+You can find Saleor App Store under the _Extensions_ tab in the Dashboard menu.
```

Comment:
No you cannot, they are called now "extensions" so i would say "You can explore available extensions under the explore section under the Extensions position in the Dashboard menu"